templates/zerver/help/view-a-messages-edit-history.md

```diff
@@ -1,13 +1,27 @@
 # View a message's edit history
 
-By default, users can view the edit history of any edited message. Edited
-messages have `EDITED` written in gray either above or to the left of the
-message.
+By default, users can view the edit history of any edited message.
+
+Messages with any content edits are labeled **EDITED** (either
+above or to the left of the message), while messages that have only been [moved to a new stream](/help/move-content-to-another-stream)
+or had a [topic edit](/help/move-content-to-another-topic) are
+labeled **MOVED** (either above or to the the left of the message).
```

Comment:
Let's factor out the `(either above or to the the left of the message)` part to a separate sentence, since there's a lot going on here.

Reply:
Also removed from instructions block.